modules/dockerregistry/tools/tools.go

```diff
@@ -0,0 +1,11 @@
+//go:build tools
```

Comment:
Please take a look at #1303, which removed the dependency with gotestsum at the package dependency level: it's now a responsibility of our CI to install it, simplifying our dependency tree.

Therefore, this file should be removed